lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -101,6 +101,7 @@
   }
   helloDate();
   eval('Date.now()'); // FAIL
+  new Function('Date.now()')() // FAIL
```

Comment:
I believe thats because smokehouse is only checking true/false right now. Need googlechrome/lighthouse/issues/964